### PR TITLE
Feat(CORE-523): style fixes for both light and dark theme

### PR DESCRIPTION
**Pull request by dshuvalov**
# Feature
- [feat: add text color in the focus-within state for outlined input in light theme](https://github.com/dialectlabs/react/commit/3ceab2eb2759ec0f36621fef94fc595458ce6df9)
## Before
Снимок экрана 2022-10-27 в 16 20 36

## After
Снимок экрана 2022-10-27 в 16 19 42

- [feat: improve hover state for buttons](https://github.com/dialectlabs/react/commit/aca5575d8a5872e1fd3e86f3045291ac9397e832). Before we are using opacity, which affects text color. I changed it to background-color opacity

## Before

https://user-images.githubusercontent.com/16071062/198295209-7d94349b-8361-4fa3-97ae-982c831c3cdc.mov

## After
https://user-images.githubusercontent.com/16071062/198294488-ec9645ac-7309-45da-b8c2-30df427ff11b.mov

- feat: add CSS overscroll-behavior
- feat: improve layout and scroll area for notification's modal

https://user-images.githubusercontent.com/16071062/198395799-6af0f7fd-3152-4d62-aae2-77d737951f80.mov

# Bug Fixes
- [fix: add apos as html code](https://github.com/dialectlabs/react/commit/f9ac7e1ee310ff28624c3671ced7f7c6e96479c0)
- [fix: loader color for notification settings in light theme](https://github.com/dialectlabs/react/commit/1af05910e5ed5b0da584e86ebc049b97885cc7ec) 

https://user-images.githubusercontent.com/16071062/198293238-d6f131e7-c048-4796-9d39-8cc77df92ea7.mov

https://user-images.githubusercontent.com/16071062/198293530-283256eb-7e84-4c1a-9cb6-d41bac01ad4e.mov

- [fix: make !dt-text-white for adornmentButton in dark theme](https://github.com/dialectlabs/react/commit/4f16806db6bfe18175d44541a66b78da029fbbbd)
## Before
Снимок экрана 2022-10-27 в 16 21 53

## After
Снимок экрана 2022-10-27 в 16 22 15

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
   },
   "scripts": {
     "bootstrap": "yarn build:all",
-    "build:all": "yarn run build:react-sdk; yarn run build:react-ui; yarn run build:react-sdk-blockchain-aptos; yarn run build:react-sdk-blockchain-solana",
+    "build:all": "yarn run build:react-sdk && yarn run build:react-ui && yarn run build:react-solana && yarn run build:react-aptos",
```
Comment (zhelezkov) [inline package.json:12]: I agree with `&&` between commands, but renamed commands doesn't looks correct ;)

Screenshot 2022-10-27 at 16 30 07

File: packages/dialect-react-ui/components/Notifications/screens/NewSettings/Wallet.tsx [hunk 2]
```diff
@@ -35,7 +35,7 @@ const Wallet = ({
   const {
     wallet: { address: walletAddress },
   } = useDialectSdk();
-  const { textStyles, outlinedInput, adornmentButton, icons, colors } =
+  const { textStyles, outlinedInput, input, adornmentButton, icons, colors } =
```
Comment (zhelezkov) [inline packages/dialect-react-ui/components/Notifications/screens/NewSettings/Wallet.tsx:38]: Screenshot 2022-10-27 at 16 31 02

**Review by zhelezkov — CHANGES_REQUESTED** [review]
overall looks good, let's get the build issues fixed.
And one comment about light style, can we have selected input field something like this? (black background looks a bit odd)
I haven't found exact widget in our figma(it's a bit unorganized sorry), but here something similar https://www.figma.com/file/q7LmGt67VqGARthmZY46rR/%5Bmain%5D-Dialect-%2F-web-notifications-widget?node-id=220%3A2050

Screenshot 2022-10-27 at 16 34 41

**Comment by dshuvalov** [comment]
> overall looks good, let's get the build issues fixed. And one comment about light style, can we have selected input field something like this? (black background looks a bit odd) I haven't found exact widget in our figma(it's a bit unorganized sorry), but here something similar https://www.figma.com/file/q7LmGt67VqGARthmZY46rR/%5Bmain%5D-Dialect-%2F-web-notifications-widget?node-id=220%3A2050
> 
> Screenshot 2022-10-27 at 16 34 41

No Problem, agree with you

**Comment by dshuvalov** [comment]
> > overall looks good, let's get the build issues fixed. And one comment about light style, can we have selected input field something like this? (black background looks a bit odd) I haven't found exact widget in our figma(it's a bit unorganized sorry), but here something similar https://www.figma.com/file/q7LmGt67VqGARthmZY46rR/%5Bmain%5D-Dialect-%2F-web-notifications-widget?node-id=220%3A2050
> > Screenshot 2022-10-27 at 16 34 41
> 
> No Problem, agree with you

@zhelezkov Fixed

https://user-images.githubusercontent.com/16071062/198335429-c79386c3-8ab3-4341-951b-04345e710871.mov


https://user-images.githubusercontent.com/16071062/198335460-4688aefd-4d96-4955-880c-1fa4cb6709ca.mov

File: package.json (hunk 1 above)
Comment (dshuvalov) [reply package.json:12]: Sorry. My fault. Fixed

File: packages/dialect-react-ui/components/Notifications/screens/NewSettings/Wallet.tsx (hunk 2 above)
Comment (dshuvalov) [reply packages/dialect-react-ui/components/Notifications/screens/NewSettings/Wallet.tsx:38]: Fixed

**Review by zhelezkov — APPROVED** [review]
this is fantastic, one more thing I have noticed at inbox example, otherwise gtg
approving in advance

Screenshot 2022-10-27 at 19 08 12

**Comment by dshuvalov** [comment]
> this is fantastic, one more thing I have noticed at inbox example, otherwise gtg approving in advance
> 
> Screenshot 2022-10-27 at 19 08 12

@zhelezkov Fixed

https://user-images.githubusercontent.com/16071062/198373047-2e27d1de-0934-4c93-acff-cfed36c4af5c.mov

**Pull request merged**